### ISSUE TITLE
transform tests to pact.io

### ISSUE THREAD

**Issue by Talank** [owncloud/client#8791]
It would be nice to start by running contract tests in the desktop client. Later, those contracts can be run in drone against both oC10 and oCIS backend.

**Comment by phil-davis** [comment]
This is another issue that the stale-bot has marked. We need #9402 to be in master.

**Comment by phil-davis** [comment]
@Talank is this something that might happen? Should it stay in the backlog project?

**Comment by Talank** [comment]
I don't think we will start working on this one soon. We need to fix GUI tests in master before looking into the pact tests. So, may be it can be removed from backlog as well.

**Comment by phil-davis** [comment]
OK - I will think about how to keep track of all the QA-team test automation issues that don't make it to the main Backlog column.
